File: AirsofterAPI/AirsofterAPI/DTO/Games/GameDetailDto.cs
namespace AirsofterAPI.DTO.Games
{
    public class GameDetailDto
    {
        public Guid Id { get; set; }
        public required string FieldName { get; set; }
        public string? CompanyName { get; set; }
        public string? ProvinceName { get; set; }
        public string? CountryName { get; set; }
        public string? Description { get; set; }
        public int CurrentPlayers { get; set; }
        public int MaxPlayers { get; set; }
        public DateTime GameDateTime { get; set; }
        public bool IsAM { get; set; }
        public List<string>? Players { get; set; }
    }
}

File: AirsofterMobile/app/src/main/java/com/airsofter/airsoftermobile/core/model/GameDetailDto.kt
package com.airsofter.airsoftermobile.core.model

data class GameDetailDto(
    val id: String,
    val fieldName: String,
    val companyName: String?,
    val provinceName: String,
    val countryName: String?,
    val description: String?,
    val currentPlayers: Int,
    val maxPlayers: Int,
    val gameDateTime: String,
    val isAM: Boolean,
    val players: List<String>?,
    val playerIds: List<String>?
)
